tests/integration/test.md5rev.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+var adapters = ['local'];
+
+adapters.forEach(function (adapter) {
+
+  describe('test.md5rev.js-' + adapter, function () {
+
+    var dbs = {};
+
+    beforeEach(function () {
+      dbs.name1 = testUtils.adapterUrl(adapter, 'testdb');
+      dbs.name2 = testUtils.adapterUrl(adapter, 'testdb2');
+    });
+
+    afterEach(function (done) {
+      testUtils.cleanup([dbs.name1, dbs.name2], done);
+    });
+
+    it('useMd5Rev=true so revision for two docs that are the same will be equal', function () {
+      const doc = {
```

Comment:
We dont use es6 in pouch yet, havent looked at the test failures but imagine this could be a problem so think its best we stick to `var` for the moment